django_elastipymemcache/client.py

```diff
@@ -1,7 +1,7 @@
 import logging
-from distutils.version import StrictVersion
 
 from django.utils.encoding import smart_str
+from packaging.version import parse
```

Comment:
distutil is deprecated in Python 3.12
https://github.com/pypa/packaging/issues/520